bylaws.md

```diff
@@ -407,68 +407,72 @@ budget of the corporation (including the budget of any committee or
-receiving the third highest number of votes shall be designated Cohort
-C directors.
+beginning after the 2017 election of directors. 
 
 Each at-large director shall hold office for the term for 
 which he or she is elected and until his or her successor shall have 
 been elected and qualified or until his or her earlier resignation, 
 removal or death.
 
-Upon completion of the term beginning after the 2017 elections, 
-directors shall be elected for a three-year term, unless they are 
+Directors shall be elected for a three-year term, unless they are 
 replacing a director that resigned or was removed, in which case such 
 replacement directors shall be elected to a term sufficient to
 complete a three-year term as measured from the term of the original
 cohort. Replacement directors shall be chosen in order of the number
 of votes received, with the longest terms of service being allocated
 to candidates according to the number of votes received.
 
-**Section 5.5.1 Election and Term of Officer Directors.** The
-corporation shall have up to two Officer Directors selected from the 
-Officers of the Corporation, with the initial selection of Officer 
-Directors corresponding to the positions of Director of Operations 
-and General Counsel, subject to a vote of the members. Officer 
-directors shall be required to maintain their Officer positions for 
-the duration of their term as Officer Directors on the Board of 
-Directors and shall be subject to approval votes for retention.
-Approval votes for the retention of Officer Directors shall be held 
-concurrent with the election of other directors on a schedule deemed 
-convenient by the Board, but with no requirement to be more frequent 
-than the election of Cohort A directors. Officer Directors can also be
-removed subject to a vote of no confidence by the at-large Directors.
-In all other respects Officer Directors shall be as at-large Directors.
+**Section 5.5.1 Selection and Retention of Appointed Directors.** 
+One appointed director position shall be reserved for the Executive 
+Director of the Corporation. Other than the position reserved for the
+Executive Director, the Board of Directors may designate as many
+additional seats as it sees fit to be appointed directors as long as 
+the majority of the seats on the Board of Directors are reserved for 
+at-large directors. Appointed directors shall be chosen to fill a
+particularized need identified by the Board of Directors, and those
+appointed must have the necessary qualification(s) to act in the 
+identified role. 
+
+Appointed directors may also be required to act as Officers of the 
+Corporation, in which case such directors shall be required to maintain 
+their Officer positions for the duration of their term as appointed 
+directors on the Board of Directors.
+
+Appointed directors (other than the Executive Director) shall serve 
+for a term of years to be identified by the Board of Directors. If
```

Comment:
Is the intention here that the board picks a term per-appointment? If so we may want to make that more clear.